### ISSUE TITLE
Getting Fatal Error

### ISSUE THREAD

**Issue by hogash** [TemplateInvaders/ti-woocommerce-wishlist#19]
Hey guys,

I'm getting this error (in php error log) when editing a page in the backend. 

```
[23-Dec-2019 09:07:01 UTC] PHP Fatal error:  Uncaught Error: Call to a member function get() on null in ../plugins/woocommerce/includes/wc-notice-functions.php:131
Stack trace:
#0 ../plugins/ti-woocommerce-wishlist/templates/ti-wishlist-empty.php(16): wc_print_notices()
#1 ../plugins/woocommerce/includes/wc-core-functions.php(249): include('/Users/mariusho...')
#2 ../plugins/ti-woocommerce-wishlist/tinv-wishlists-function.php(141): wc_get_template('ti-wishlist-emp...', Array, '')
#3 ../plugins/ti-woocommerce-wishlist/public/wishlist/view.class.php(747): tinv_wishlist_template('ti-wishlist-emp...', Array)
#4 ../plugins/ti-woocommerce-wishlist/public/wishlist/view.class.php(674): TInvWL_Public_Wishlist_View->wishlist_empty(Array, Array)
#5 .. in ../plugins/woocommerce/includes/wc-notice-functions.php on line 131
```

From my findings, replacing this line"

```
<?php if ( function_exists( 'wc_print_notices' ) ) { wc_print_notices(); } ?>
```

in `ti-wishlist-empty.php(16)` with this:

```
<?php if ( function_exists( 'wc_print_notices' ) && function_exists('WC') && WC()->session ) { wc_print_notices(); } ?>
```

fixes the error.

**Comment by doozy** [comment]
Hi @hogash 

Thanks for the bug report.

It was fixed in the https://github.com/TemplateInvaders/ti-woocommerce-wishlist/commit/82e0771a5e1ff53ffd0804ad43bfb170dfae52fa

We will release the official update soon.

**Comment by doozy** [comment]
Released in version 1.16.0
https://github.com/TemplateInvaders/ti-woocommerce-wishlist/releases/tag/1.16.0
